release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -175,6 +175,29 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
 [id="ocp-4-10-web-console"]
 === Web console
 
+[id="ocp-4-10-web-console-developer-perspective"]
+==== Developer perspective
+* With this update, you can specify the name of a service binding connector in the *Topology* view while making a binding connection.
+* With this update, creating pipelines workflow has now been enhanced:
+** You can see when the pipeline has been created but not started by looking at the status of the pod in the *Topology* view.
+** You can create an application using the *Import from Git* workflow, along with a template-generated pipeline by selecting the *Add pipeline* option. The webhook resources and URL are visible in the side panel of the selected resources.
```

Comment:
```suggestion
**You can now choose an user defined pipeline from a dropdown list while importing your application from import from Git/Docker pipeline workflow.
**Default webhooks are added for the pipelines that are created using Add flow and URL is visible in the side panel of the selected resources.
```

Reply:
Thanks for the suggestion @karthikjeeyar